### ISSUE TITLE
Core dumped

### ISSUE THREAD

**Issue by ilhamm02** [iron-fish/ironfish#538]
After mining about 6 hours getting an unknown error

```<--- Last few GCs --->

[175:0x7fdcd4000ec0] 46250373 ms: Scavenge 353.6 (403.6) -> 337.9 (403.6) MB, 74.6 / 0.0 ms  (average mu = 0.986, current mu = 0.986) allocation
failure
[175:0x7fdcd4000ec0] 46250758 ms: Scavenge 353.6 (403.6) -> 337.9 (403.6) MB, 8.2 / 0.0 ms  (average mu = 0.986, current mu = 0.986) allocation f
ailure
[175:0x7fdcd4000ec0] 46251534 ms: Scavenge 353.6 (403.6) -> 337.9 (403.6) MB, 3.9 / 0.0 ms  (average mu = 0.986, current mu = 0.986) allocation f
ailure


<--- JS stacktrace --->

FATAL ERROR: MarkCompactCollector: young object promotion failed Allocation failed - JavaScript heap out of memory
 1: 0xb02ec0 node::Abort() [node]
 2: 0xa181fb node::FatalError(char const*, char const*) [node]
 3: 0xced88e v8::Utils::ReportOOMFailure(v8::internal::Isolate*, char const*, bool) [node]
 4: 0xcedc07 v8::internal::V8::FatalProcessOutOfMemory(v8::internal::Isolate*, char const*, bool) [node]
 5: 0xea5ea5  [node]
 6: 0xed4668 v8::internal::EvacuateNewSpaceVisitor::Visit(v8::internal::HeapObject, int) [node]
 7: 0xef3e6e v8::internal::FullEvacuator::RawEvacuatePage(v8::internal::MemoryChunk*, long*) [node]
 8: 0xece2cf v8::internal::Evacuator::EvacuatePage(v8::internal::MemoryChunk*) [node]
 9: 0xece5eb v8::internal::PageEvacuationJob::Run(v8::JobDelegate*) [node]
10: 0x18da9e4 v8::platform::DefaultJobWorker::Run() [node]
11: 0xb6f2f5  [node]
12: 0x7fdf83696ea7  [/lib/x86_64-linux-gnu/libpthread.so.0]
13: 0x7fdf835c6def clone [/lib/x86_64-linux-gnu/libc.so.6]
/usr/src/app/bin/ironfish: line 45:   175 Aborted                 (core dumped) "$NODE" "$DIR/run" "$@"```

**Comment by dguenther** [comment]
Could you retest on version 0.1.9 and see if you're still running into this issue?

**Comment by dguenther** [comment]
Closing this issue as it should have been fixed in 0.1.9 -- Please reopen an issue if you're still seeing this problem.
